Classes/Public/Metrics.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
```

Comment:
👍  I didn't notice the Constants folder after I merged develop.  Def belongs in there.